src/python/grapl_analyzerlib/grapl_analyzerlib/BUILD

```diff
@@ -1,10 +1,5 @@
 python_library(
-    dependencies=[
-        # TODO: Ideally, this would be picked up by Pants' automatic
-        # dependency resolution, but something appears to be off with
-        # it.
-        "src/python/graplinc/graplinc/grapl/api"
-    ],
+    dependencies=[],
```

Comment:
I think this kwarg can go away completely

Reply:
yep i noticed that :)